src/ide/editor/add-value.tsx

```diff
@@ -1,13 +1,14 @@
 import {observer} from "mobx-react";
 import * as React from "react";
-import {indexOf} from "lodash";
 
 import {editorState, FocusType} from "./state";
 import {preventBubbleUp} from "./utils/ui-util";
 
-import {default as metaModelInstance} from "../../meta/meta-model";
+import {default as metaModelInstance} from "../instance";
 const sTypes = metaModelInstance.sTypes();
 
+const styles = require("./styles.scss");
```

Comment:
Cool, ha? 💃 :D  But t mightt be some styles are not being applied due to being injected from js, not sure about that. Would be great if you can double check that with `npm run ide` :wink: